analysis.py

```diff
@@ -244,7 +246,7 @@ def fill_rows(content: List[Content], node_logs: List[str]) -> Dict[str, List[An
         task_body = task.json["task"].split(":", 1)
         task_type = task_body[0].replace("<", "").strip()
         task_desc = task_body[1].replace(">", "").strip()
-        task_body_json = yaml.safe_load(task_desc.replace("'", '"'))
```

Comment:
Why do we need this?

Reply:
We don't, that was a leftover from when it was parsed by the `json` parser.